misk-hibernate/src/main/kotlin/misk/hibernate/RealTransacter.kt

```diff
@@ -135,7 +135,7 @@ internal class RealTransacter private constructor(
     }
   }
 
-  override fun retries(): Transacter = withOptions(options.copy(maxAttempts = 2))
+  override fun retries(maxAttempts: Int): Transacter = withOptions(options.copy(maxAttempts = maxAttempts))
```

Comment:
how about we set the default to 2 to maintain compatibility?

Reply:
Sure why not